azavea/raster-foundry
# Allow deleting scenes if scene owner or in root org

## Overview
Hook up UI to allow deleting imports
Clean up scene-item component

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

## Testing Instructions

* Delete a scene from the imports page and verify that it works
* make a delete request to a scene not owned by your user, and verify that it doesn't work unless you are in the root organization
* Verify that failed deletions for a) scenes not visible by user b) scenes not found and c) scenes which the user does not have permission to delete have the correct error codes (404, 404, and 403)

Closes #1908
